controllers/articles.controllers.js

```diff
@@ -41,3 +42,22 @@ exports.getCommentByArticleId = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.postNewComment = (req, res, next) => {
+    const {article_id} =req.params
+    const newComment = req.body
+    const commentPromises = [checkArticleExists(article_id)];
```

Comment:
check to see article exists isn't really necessary.  You can just throw a custom error. An empty array coming back in rows in your model will mean that article doesn't exist. No need to change though